src/chatClient.cpp

```diff
@@ -1025,6 +1064,7 @@ promise::Promise<void> Client::connect(Presence pres, bool isInBackground)
     }
 
     assert(mConnState == kDisconnected);
+    mInitStats.stageEnd(InitStats::kStatsPostFetchNodes);
```

Comment:
This line should go immediately before the `mInitStats.stageStart(InitStats::kStatsConnection);`